test/services/todo.test.js

```diff
@@ -0,0 +1,127 @@
+'use strict'
+
+const { test } = require('tap')
+const { build, cleandb } = require('../helper')
+
+test('should create an item', async (t) => {
+  const app = build(t)
+  t.teardown(cleandb)
```

Comment:
It's better to clean the db _before_ each test, not after. I would put in a 
beforeEach in tap.

Reply:
Thanks will do. For travis, I will first transfer the repo to org first then add it (I remember it was painful to set it up when transferred)